README.aspnet.md

```diff
@@ -117,5 +117,5 @@ See the [.NET Framework Lifecycle FAQ](https://support.microsoft.com/help/17455/
 # License
 
 * Legal Notice: [Container License Information](https://aka.ms/mcr/osslegalnotice)
```

Comment:
How about this instead:

* The [MICROSOFT SOFTWARE SUPPLEMENTAL LICENSE FOR WINDOWS CONTAINER BASE IMAGE](https://hub.docker.com/_/microsoft-windows-servercore/) applies to .NET Framework container images.
* The [Visual Studio Tools license](https://visualstudio.microsoft.com/license-terms/mlt031519/) applies to [.NET Framework SDK images](https://hub.docker.com/_/microsoft-dotnet-framework-sdk/).
* [General legal notice for Microsoft container images](https://aka.ms/mcr/osslegalnotice)

Reply:
We could then do the same thing for the ~/dotnet README (minus the VS one).